csv2bufr/cli.py

```diff
@@ -136,21 +135,21 @@ def transform(ctx, csv_file, mapping, output_dir, station_metadata,
         with open(filename, "wb") as fh:
             fh.write(result[item].read())
 
-        # convert to JSON, ideally we would do this from in memory object
-        # but I can't figure out how to do this with eccodes.
-        fh = open(filename, "rb")
-        handle = codes_bufr_new_from_file(fh)
-        fh.close()
-
-        result[item].seek(0)
-        if template is not None:
-            json_dict = bufr_to_json(handle, template)
-            json_dict["md5sum"] = item
-            json_dict["bufr_string_b64"] = \
-                base64.b64encode(result[item].read()).decode("utf-8")
-            filename = f"{output_dir}{os.sep}{item}.json"
-            with open(filename, "w") as fh:
-                json.dump(json_dict, fh, indent=2)
+        # convert to JSON if template specified
+        if json_template is not None:
+            click.echo("Writing JSON data to file")
+            # read from BUFR file, ideally we would do this from in memory object
+            # but I can't figure out how to do this with eccodes.
+            with open(filename, "rb") as fh:
+                handle = codes_bufr_new_from_file(fh)
+            json_dict = bufr_to_json(handle, json_template)
```

Comment:
addressing comment

can we create the GeoJSON as a Python dict instead, which then gets serialized via json.dumps?

Reply:
Ah, so eccodes serializes JSON via a template.